Penguins_chapter.qmd

```diff
@@ -0,0 +1,1007 @@
+---
+title: "Exploring data on penguins" 
+author: "CHOULETTE Klervi, DENIS Manon, DUPORT Maureen, MOTTE Arthur, POYARD Antoine, SESIA Marion, TANGUY Céline" 
+bibliography: references.bib 
+execute:
+  freeze: auto
+  warning: false
+output: 
+  html_document: 
+    toc: true 
+    toc_float: true 
+    toc_depth: 3
+---
+
+# Introduction {style="color:navy"}
+
+## Study context {style="color:royalblue"}
+
+This document is a report on the analysis of a dataset on penguins. The data collected contains informations on multiples [variables](#colum-name). The dataset was find on Kaggle : https://www.kaggle.com/datasets/utkarshx27/penguin-size-clutch-and-blood-isotope-data?select=penguins_raw.csv. This dataset was made from differents articles according by Gorman *et al.* [@10.1371/journal.pone.0090081] and Horst *et al.* [@10.32614/RJ-2022-020]. It focused on three penguin species : Adelie (*Pygoscelis adeliae*), Chinstrap (*Pygoscelis antarcticus*) and Gentoo (*Pygoscelis papua*). The dataset contains 344 observations and 17 variables. The goal of this analysis is to explore the dataset and to identify the main characteristics of the penguins.
```

Comment:
Little conjugation error : "was found" instead of "was find"

Reply:
Noted, we're going to change that